app/components/jobseekers/school_group_overview_component.html.haml

```diff
@@ -13,13 +13,19 @@
           %td.govuk-table__cell{ class: "govuk-!-font-weight-bold" }
             = t('school_groups.website')
           %td.govuk-table__cell
-            = link_to(t('jobs.website_link_text', organisation_name: @vacancy.parent_organisation.name), @vacancy.parent_organisation.website.presence || @vacancy.parent_organisation.url, class: 'govuk-link wordwrap', target: '_blank')
+            = link_to(t('schools.website_link_text', organisation_name: @vacancy.parent_organisation.name), @vacancy.parent_organisation.website.presence || @vacancy.parent_organisation.url, class: 'govuk-link wordwrap', target: '_blank')
       - if @vacancy.contact_email.present?
         %tr.govuk-table__row
           %td.govuk-table__cell{ class: "govuk-!-font-weight-bold" }
-            = t('school_groups.email')
+            = t('jobs.contact_email')
           %td.govuk-table__cell
             = mail_to @vacancy.contact_email, @vacancy.contact_email, class: 'govuk-link link-wrap', subject: t('jobs.contact_email_subject', job: @vacancy.job_title), body: t('jobs.contact_email_body', url: job_url(@vacancy))
+      - if @vacancy.contact_number.present?
+        %tr.govuk-table__row
+          %td.govuk-table__cell{ class: "govuk-!-font-weight-bold" }
+            = t('jobs.contact_number')
+          %td.govuk-table__cell
+            = link_to @vacancy.contact_number, "tel:#{@vacancy.contact_number}", class: 'govuk-link'
```

Comment:
Where does `tel:` come from? Are you rendering 'tel:01234567890' to the page, or is `tel:` a `link_to` helper method that formats the number?

Reply:
https://tools.ietf.org/html/rfc3966